.github/workflows/build-docs.yaml

```diff
@@ -8,6 +8,12 @@ on:
 
 jobs:
   build-docs:
+    strategy:
+      matrix:
+        python-version:
+        - "3.10"  # Earliest version supported by ixmp4
```

Comment:
Building the docs only needs to be checked with the version that is actually used on readthedocs.

Reply:
Yes...

https://github.com/iiasa/ixmp4/blob/27b3000facd3fa96b278ebe471e429d98ee64c16/.readthedocs.yaml#L12

Reply:
And do we want to keep that or try the more future-proof version?

Reply:
Fine to keep this until 3.10 isn't supported any more by ixmp4 - my point was just to minimize the computational load.